src/bundle/Resources/public/js/alloyeditor/src/buttons/ez-btn-anchoredit.js

```diff
@@ -87,7 +87,7 @@ export default class EzBtnAnchorEdit extends Component {
 
             return (
                 value &&
-                [...container.querySelectorAll(`#${value}`)].every((element) => {
+                [...container.querySelectorAll(`[id="${value}"`)].every((element) => {
```

Comment:
This is a good candidate for a comment on why it's done by `[id]` and no with `#`. In one year someone might be surprised why it is like this.